.github/scripts/filter_changed_files_go_test.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -euo pipefail
```

Comment:
This won't solve the underlying problem, but it should guard against a silent failure